apollo-router/src/plugins/connectors/make_requests/request_merger.rs

```diff
@@ -0,0 +1,171 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use apollo_compiler::collections::HashSet;
+use apollo_compiler::collections::IndexMap;
+use apollo_federation::sources::connect::Namespace;
+use http::response::Parts;
+use serde_json::Value as JsonValue;
+use serde_json_bytes::ByteString;
+use serde_json_bytes::Map;
+use serde_json_bytes::Value;
+use serde_json_bytes::json;
+
+use crate::Context;
+use crate::plugins::connectors::make_requests::RequestInputs;
+use crate::services::external::externalize_header_map;
+
+pub(crate) struct RequestMerger<'merger> {
```

Comment:
i'm wondering if this should be `VariableMerger` since it's also used for response mapping. Or I sometimes want to call it `MappingContext`.

Reply:
4c48d58ed